src/org/traccar/notification/NotificationMail.java

```diff
@@ -43,7 +43,7 @@ private static Properties getProperties(PropertiesProvider provider) {
         if (host != null) {
             properties.put("mail.transport.protocol", provider.getString("mail.transport.protocol", "smtp"));
             properties.put("mail.smtp.host", host);
-            properties.put("mail.smtp.port", provider.getString("mail.smtp.port", "25"));
+            properties.put("mail.smtp.port", String.valueOf(provider.getInteger("mail.smtp.port", 25)));
```

Comment:
According to documentation it should be integer but stored as string https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html

> The SMTP protocol provider supports the following properties, which may be set in the JavaMail Session object. The properties are always set as strings; the Type column describes how the string is interpreted. For example, use
        props.put("mail.smtp.port", "888");
to set the mail.smtp.port property, which is of type int.